Cargo.toml

```diff
@@ -24,6 +24,7 @@ futures = "0.3"
 rlimit = "0.10.1"
 log = "0.4.0"
 env_logger = "0.11.3"
+anstream = "=0.6.14"
```

Comment:
👀  what's this for?

Reply:
Got this error when running tests:
```bash
cargo test 
error: failed to select a version for `env_logger`.
    ... required by package `rustscan v2.2.4-rc1 (/Users/leofvo/Developer/opensource/RustScan)`
versions that meet the requirements `^0.11.3` (locked to 0.11.3) are: 0.11.3

the package `rustscan` depends on `env_logger`, with features: `anstream` but `env_logger` does not have these features.
 It has an optional dependency with that name, but that dependency uses the "dep:" syntax in the features table, so it does not have an implicit feature with that name.


failed to select a version for `env_logger` which could resolve this conflict
```

So the way I fixed was by adding the dependency explicitly here :/